### PR TITLE
Using dice as loss function

### PR DESCRIPTION
**Pull request by FranciscoRibeiro**
The training script now uses dice coefficient for the loss function calculations.
Set prediction batch size (application.py) to 1 because there's frequently very little memory available.
Model trained with dice was added.
Check commit messages for more details.

### CODE REVIEW

File: skull_extract.py [hunk 1]
```diff
@@ -54,25 +52,40 @@ def dice_coef_loss(y_true, y_pred):
     mris = np.rollaxis(mris, 2).reshape(mris.shape[2], 176, 256, 1)
     masks = np.rollaxis(masks, 2).reshape(masks.shape[2], 176, 256, 1)
 
+    # Split data for training and
```
Comment (dinocosta) [inline skull_extract.py]: This comment ran out of breath haha @FranciscoRibeiro

**Review by dinocosta — APPROVED** [review]
